ci/jenkins/Build.groovy.j2

```diff
@@ -259,6 +259,25 @@ stage('Build') {
       Utils.markStageSkippedForConditional('BUILD: RISC-V')
     }
   },
+  'BUILD: CSINN2': {
+    if (!skip_ci && is_docs_only_build != 1) {
+      node('CPU-SMALL') {
+        ws({{ m.per_exec_ws('tvm/build-csinn2') }}) {
+          docker_init(ci_riscv)
+          init_git()
+          sh (
+            script: "${docker_run} ${ci_riscv} ./tests/scripts/task_config_build_csinn2.sh build-csinn2-x86 &&
+                     ${docker_run} ${ci_riscv} ./tests/scripts/task_config_build_c906.sh build-csinn2-c906",
+            label: 'Create CSINN2 x86 and c906 cmake config',
+          )
+          make(ci_riscv, 'build-csinn2-x86', '-j2')
+          make(ci_riscv, 'build-csinn2-c906', 'tvm_rpc -j2')
```

Comment:
I pushed your changes to a branch in apache/tvm since that's the only way to test these Jenkinsfile changes, it looks like there is a problem with the newline here: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/csinn_jenkins/1/pipeline/

I think this would fix it

```suggestion
            script: "${docker_run} ${ci_riscv} ./tests/scripts/task_config_build_csinn2.sh build-csinn2-x86",
            label: 'Create CSINN2 x86 cmake config',
          )
          make(ci_riscv, 'build-csinn2-x86', '-j2')
          sh (
            script: "${docker_run} ${ci_riscv} ./tests/scripts/task_config_build_c906.sh build-csinn2-c906",
            label: 'Create CSINN2 c906 cmake config',
          )
          make(ci_riscv, 'build-csinn2-c906', 'tvm_rpc -j2')
```